src/ZulipMobile.js

```diff
@@ -21,17 +21,17 @@ if (config.enableSentry) {
 }
 
 export default () => (
-  <StoreProvider>
-    <TranslationProvider>
-      <StylesProvider>
-        <CompatibilityChecker>
-          <AppEventHandlers>
-            <AppDataFetcher>
+  <CompatibilityChecker>
```

Comment:
If app is not compatible, then why should we load/render data.
And all the events/data are inside it, so internal change will not recall it.